packages/cli/generators/controller/index.js

```diff
@@ -127,6 +127,16 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
             default: modelList[0],
             validate: utils.validateClassName,
           },
+          {
+            type: 'input',
+            name: 'modelNamePlural',
+            message:
+              'What is the custom plural form of the model (for building REST URL)',
```

Comment:
As I commented above, please rephrase this question to make it clear that we are asking for the URL base path.